babel.config.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+module.exports = {
+  presets: ['@babel/preset-react', '@babel/preset-flow'],
```

Comment:
Can we remove these presets and use the plugins directly? That way expected stuff can't sneak in.